File: backend/src/main/java/com/example/shoestore/dto/request/AddressRequest.java
package com.example.shoestore.dto.request;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class AddressRequest {
    private Integer provinceId;
    private String provinceName;
    private Integer districtId;
    private String districtName;
    private Integer wardId;
    private String wardName;
    private String streetDetail;
    private String userName;
    private String phone;
}

File: android/app/src/main/java/com/example/shoeshop/dto/request/AddressRequest.kt
package com.example.shoeshop.dto.request

data class AddressRequest(
    val provinceId: Int,
    val provinceName: String,
    val districtId: Int,
    val districtName: String,
    val wardId: Int,
    val wardName: String,
    val streetDetail: String,
    val userName: String,
    val phone: String
)